### PR TITLE
Hypospray nerf: Normal ones only inject up to 10, cmo's injects up to 20

### PR DESCRIPTION
**Pull request by silicons**
should move it to its own var instead of using transfer amounts on the hypovials but I'm lazy so webeditor pr

### CODE REVIEW

**Review by deathride58 — DISMISSED** [review]

File: modular_citadel/code/modules/reagents/reagent container/hypovial.dm [hunk 1]
```diff
@@ -68,11 +68,11 @@
 /obj/item/reagent_containers/glass/bottle/vial/small
 	name = "hypovial"
 	volume = 60
```
Comment (Poojawa) [inline modular_citadel/code/modules/reagents/reagent container/hypovial.dm:70]: > changes volume to being insane

File: modular_citadel/code/modules/reagents/reagent container/hypovial.dm [hunk 2]
```diff
@@ -68,11 +68,11 @@
...
-	possible_transfer_amounts = list(5,10,15,30)
```
Comment (Poojawa) [inline modular_citadel/code/modules/reagents/reagent container/hypovial.dm:71]: > suddenly realizes two charge deathmix is bad

🤔

File: modular_citadel/code/modules/reagents/reagent container/hypovial.dm [hunk 3]
```diff
@@ -82,7 +82,7 @@
 	desc = "A large hypovial, for deluxe hypospray models."
 	icon_state = "hypoviallarge"
 	volume = 120
```
Comment (Poojawa) [inline modular_citadel/code/modules/reagents/reagent container/hypovial.dm:84]: > CMO specific vial buff containment

File: modular_citadel/code/modules/reagents/reagent container/hypovial.dm [hunk 4]
```diff
@@ -82,7 +82,7 @@
...
-	possible_transfer_amounts = list(5,10,15,20,30,60)
+	possible_transfer_amounts = list(5,10,15,20)
```
Comment (Poojawa) [inline modular_citadel/code/modules/reagents/reagent container/hypovial.dm:85]: > Oh geeze, deathmix is too easy to put in them again.

🤔

**Review by deathride58 — APPROVED** [review]

**Pull request merged**